src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -320,24 +318,42 @@ def test_unmoveable_object(self):
         body = response.text
         self.assertIn("cannot be moved", body)
 
-    @unittest.skipIf(str is not bytes, #Py2 only
-                     "Only Python 2 can do str(b'encoded') and get the right thing")
     def test_copy_then_delete_with_unicode_name(self):
         # Tests unicode on object copied then deleted (#238579)
-        # The zope.publisher.browser conversion methods for text/str
-        # rely on the default encoding, which breaks on this unicode name in Python 3.
-        # We either wind up with "b'voil\\xe0'" or a list of the ints that make
-        # up the bytes, depending on which order we try for the type names.
+
+        # The 'Accept-Charset' is important on Python 3 to get
```

Comment:
Oh dear, now I have flashbacks about my old Zope 2 personal website breaking for users of Internet Explorer because it used to send a different Accept-Charset header than my Firefox.